src/sidescript_readDEM.R

```diff
@@ -1,3 +1,9 @@
+if(!dir.exists("data/maps/alt_30s_bil")) {
```

Comment:
also here, use the `here` package

Reply:
The code has been changed using the here function.